resources/models/resource.py

```diff
@@ -262,6 +262,15 @@ class Resource(ModifiableModel, AutoIdentifiedModel):
                                                                       null=True, blank=True)
     reservable_min_days_in_advance = models.PositiveSmallIntegerField(verbose_name=_('Reservable min. days in advance'),
                                                                       null=True, blank=True)
+    cancellation_min_days_in_advance = models.PositiveIntegerField(
```

Comment:
In my experience these types of cancellations are done in hours and not days. Are you sure it should be implemented in days? Also if this is changed please remember to change the frontend as well.

Reply:
That's a good point.
In this case customer wanted it in days instead of hours. However, while calculating the difference between the reservation start datetime and cancellation datetime, the hours are taken into consideration.